sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -500,10 +519,22 @@ public partial class IndexingParameters
     {
         public IndexingParameters() { }
         public int? BatchSize { get { throw null; } set { } }
-        public System.Collections.Generic.IDictionary<string, object> Configuration { get { throw null; } set { } }
+        public System.Collections.Generic.IDictionary<string, object> Configuration { get { throw null; } }
         public int? MaxFailedItems { get { throw null; } set { } }
         public int? MaxFailedItemsPerBatch { get { throw null; } set { } }
     }
+    public static partial class IndexingParametersExtensions
+    {
+        public static Azure.Search.Documents.Indexes.Models.IndexingParameters ExcludeFileNameExtensions(this Azure.Search.Documents.Indexes.Models.IndexingParameters parameters, params string[] extensions) { throw null; }
+        public static Azure.Search.Documents.Indexes.Models.IndexingParameters IncludeFileNameExtensions(this Azure.Search.Documents.Indexes.Models.IndexingParameters parameters, params string[] extensions) { throw null; }
+        public static Azure.Search.Documents.Indexes.Models.IndexingParameters ParseDelimitedTextFiles(this Azure.Search.Documents.Indexes.Models.IndexingParameters parameters, params string[] headers) { throw null; }
```

Comment:
There is also a "Default" parse mode available on the Azure portal besides the other five you enumerated (Text, Json, JsonArrays, JsonLines, DelimitedText).  Are we exposing that?

Reply:
I can. What does it do? Note, all this code was ported from track 1 that @brjohnstmsft worked on. The only thing I changed (apart from some implementation details - see above), was the `IndexFileNameExtensions` to `IncludeFileNameExstensions`, which I'm going to rename back.

Reply:
Seems we don't need to since it is the default, but let me know if that's an incorrect assumption.